usaon_vta_survey/templates/macros/javascript_remove.j2

```diff
@@ -0,0 +1,15 @@
+{% macro add_javascript__remove() -%}
+  <script>
+    function remove(url, response) {
+      fetch(url, { method: 'DELETE'})
+          .then(response => {
+              if (response.ok && response.redirected) {
+                    location.assign(response.url)
+              } else {
+                    console.error(`Something unexpected happened when trying to delete ${response.url}`)
```

Comment:
```suggestion
                    console.error(`Something unexpected happened with query: DELETE ${response.url}: ${response.body}`)
```

I'm not sure if `response.body` is correct, but whatever attribute holds the content would be great to log. You can find this in the network tab of browser dev tools, but if you didn't have it open already that info wouldn't have been recorded, so I think it's good to `console.error` it.